libkbfs/conflict_resolver.go

```diff
@@ -3043,6 +3066,12 @@ func (cr *ConflictResolver) completeResolution(ctx context.Context,
 		return err
 	}
 
+	defer func() {
```

Comment:
Ah, that's where you added the other call to `cleanUpBlockState`! I think that suffices.